lib/pages/chat_list/chat_list_item.dart

```diff
@@ -144,8 +143,12 @@ class ChatListItem extends StatelessWidget with ChatListItemMixin {
                 ),
                 Expanded(
                   child: Column(
+                    mainAxisAlignment: MainAxisAlignment.center,
                     children: [
-                      ChatListItemTitle(room: room),
+                      ChatListItemTitle(
+                        room: room,
+                        textStyle: LinagoraTextStyle.material().bodyMedium2,
```

Comment:
why not use: `Theme.of(context).textTheme.bodyMedium,`

Reply:
<img width="481" alt="Screenshot 2024-09-11 at 11 43 48" src="https://github.com/user-attachments/assets/6eeb95bf-5bab-4626-85ce-5a76452d8210">